File: src/Python/Arrays/Questao-1.py
# Escreva um programa que cria um array bidimensional, de números inteiros e calcula a soma de todos os elementos no array.
 
array = [[1, 2, 3], [4, 5, 6], [7, 8, 9]] # Array bidimensional
soma = 0 # Variavel que vai armazenar a soma dos elementos do array

for i in array : # Para cada elemento do array
    for j in i : # Para cada elemento do array dentro do array
        soma += j # Soma o elemento atual ao valor da soma

print(soma) # Imprime a soma dos elementos do array

File: src/JavaScript/vetores/Nivel1/Questao-1.js
// Implementar subprograma que retorna a soma todos os elementos de um vetor

// Função para somar todos os elementos de um vetor
function somaVetor(array) {
    let soma = 0; // Inicializa a variável soma com zero
    for (let i = 0; i < array.length; i++) {
        soma += array[i]; // Adiciona cada elemento do vetor à soma
    }
    return soma; // Retorna a soma total
}

// Exemplo de uso da função com um vetor
const array = [1, 2, 3];
console.log(somaVetor(array)); // Imprime a soma dos elementos do vetor [1, 2, 3]
